### DISCUSSION TITLE
Image snapping to outer boundary?

### DISCUSSION THREAD

**Discussion by keepitrio** [ValentinH/react-easy-crop#441] [Q&A]
Is there a way to implement the cropper so that if I drag beyond the x or y boundary, it snaps the image so the x or y position is 0 instead of a negative number?

**Comment by ValentinH** [comment]
This is not something that is directly supported by the library. However, you are in control of the `crop` property. Therefore, when `onCropChange` is called, you can choose to ignore it if the value is outside of a specific range.
With that in mind, you can use the new crop value with the image size and container size to decide if the new value is valid or not.
